org.opent2t.sample.thermostat.superpopular/com.smartthings.thermostat/js/thingTranslator.js

```diff
@@ -279,7 +285,7 @@ class Translator {
      * Updates the specified resource with the provided payload.
      */
     postDeviceResource(di, resourceId, payload) {
-        if (di === thermostatDeviceDi)
+        if (di === generateGUID(this.controlId+'opent2t.d.thermostat'))
```

Comment:
Fixed.

---
In reply to: [108803828](https://github.com/openT2T/translators/pull/217#discussion_r108803828) [](ancestors = 108803828)